### ISSUE TITLE
Random Game from Backlog

### ISSUE THREAD

**Issue by iedame** [Clidus/gwl#80]
It would be nice to have a page to pick a random game from the backlog.

Filtering by system as optional?

**Comment by Clidus** [comment]
I like it, kinda like a "I'm feeling lucky" button?

**Comment by haoji** [comment]
Yeah, exactly

**Comment by crunchprank** [comment]
I agree, great idea. It would help me stop procrastinating choosing a game by having some random algorithm do it for me :)
